### PR TITLE
Aerospace engineering minor

### PR DESCRIPTION
**Pull request by hahnbeelee**
### Summary
I was told to add the aerospace major, but by the looks of it it looks like there is no aerospace major? So, I added the minor instead.
image

### Test Plan 
- Go through json + make sure everything looks good + logic lines up with [here](https://courses.cornell.edu/content.php?catoid=41&navoid=11649#aerospace-minor)
- Add the aerospace minor and make sure everything looks good on the site

### CODE REVIEW

File: src/requirements/data/minors/aerospace.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        'MAE 3783',
+        'ECE 2100',
+        'ENGRD 2100',
+        'PHYS 3360',
+      ]
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1, 1],
+    minNumberOfSlots: 2,
+    slotNames: ['Group B', 'Group B', 'Group C', 'Group C'],
```
Comment (SamChou19815) [inline src/requirements/data/minors/aerospace.ts:107]: Do not repeat, but instead change it to:

```ts
{
    // ...
    checker: includesWithSubRequirements(
      [
        'MAE 4020',
        'MAE 5020',
        'MAE 4130',
        'MAE 4180',
        'MAE 5180',
        'MAE 4700',
        'MAE 4770',
        'MAE 5700',
        'MAE 4730',
        'MAE 5730',
        'MAE 4780',
        'MAE 5780',
        'MAE 5130',
        'MAE 5430',
        'MAE 6510',
      ],
      [
        'ENGRD 2020',
        'MAE 2030',
        'ENGRD 2210',
        'MAE 3230',
        'MAE 3240',
        'MAE 3260',
        'MAE 3270',
        'MAE 3780',
        'MAE 3783',
        'ECE 2100',
        'ENGRD 2100',
        'PHYS 3360',
      ],
    ),
    fulfilledBy: 'courses',
    perSlotMinCount: [2,2],
    slotNames: ['Group B', 'Group C'],
}
```

File: src/requirements/data/minors/aerospace.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        'MAE 4230',
+        'MAE 5230',
+        'MAE 4510',
+        'MAE 5070',
+        'MAE 5510',
+      ]
+    ),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 3],
+    slotNames: ['MAE 3050 or MAE 4060', 'Group A'],
```
Comment (SamChou19815) [inline src/requirements/data/minors/aerospace.ts:30]: probably name it "other group a course"?

**Review by SamChou19815 — CHANGES_REQUESTED** [review]

File: src/requirements/data/minors/aerospace.ts (hunk 1 above)
Comment (hahnbeelee) [reply src/requirements/data/minors/aerospace.ts:107]: but, you can fulfill this requirement with any combination of groups:
1 from group b, 1 from group c or
2 from group b or
2 from group c

Comment (hahnbeelee) [reply src/requirements/data/minors/aerospace.ts:107]: i need like a minNumberOfCourses to make this non-repeatable

Comment (SamChou19815) [reply src/requirements/data/minors/aerospace.ts:107]: ok that makes sense

File: src/requirements/decorated-requirements.json [hunk 3]
```diff
@@ -142093,6 +142093,129 @@
...
+          "description": "Select at least four core aerospace engineering courses of which one must choose MAE 3050 or MAE 4060 (or both).",
+          "source": "https://courses.cornell.edu/content.php?catoid=41&navoid=11649#aerospace-minor",
+          "fulfilledBy": "courses",
+          "perSlotMinCount": [
+            1,
+            3
+          ],
+          "slotNames": [
+            "MAE 3050 or MAE 4060",
+            "Group A"
```
Comment (SamChou19815) [inline src/requirements/decorated-requirements.json:142113]: Regenerate the file

**Review by SamChou19815 — APPROVED** [review]
LGTM. final nit

**Pull request merged**